pixi/backend/api.js

```diff
@@ -61,6 +62,7 @@ const execChecks = async (url) => {
     redirected: res.redirected,
     url: res.url,
     isAmp: isAmp($),
+    version: API_VERSION
```

Comment:
You do not need that here, since the version is already added after the `await execChecks` below.